generator.py

```diff
@@ -54,7 +54,7 @@ def update_workspace():
 for entry in [f for f in listdir('.') if isfile(join('.', f))]:
     if entry.startswith('Gir_Gst') and entry.endswith('.toml'):
         print('==> Regenerating "{}"...'.format(entry))
-        call(['./target/release/gir', '-c', entry])
+        call(['./target/release/gir', '-c', entry, '-d', 'gir-files', '-m', 'sys'])
```

Comment:
just realize this patch is not needed since all this data is in the toml file

Reply:
Yeah, can you drop that? :)

Reply:
done ;)